Makefile

```
@@ -99,7 +99,7 @@ convert_feature_manifests:
 	git submodule init;\
 	git submodule update --recursive;\
 	cd components/support/nimbus-fml;\
-	cargo run "../../../../monitor-web.fml.yaml" experimenter --output "../../../../monitor-web.yaml" --channel developer;
+	cargo run "../../../../monitor-web.fml.yaml" experimenter --output "../../../../monitor-web.yaml" --channel beta;
```

Since we're updating this it's worth noting the `experimenter` command is removed in [this PR](https://github.com/mozilla/application-services/pull/5784) in favor of `generate-experimenter`.
```suggestion
	cargo run generate-experimenter --channel beta "../../../../monitor-web.fml.yaml" "../../../../monitor-web.yaml";
```

---

That PR hasn't landed and this issue is blocking feature manifests from updating, which in turn is blocking experiments from being configured.